rented-properties/src/main/java/org/egov/cpt/service/AccountStatementExcelGenerationService.java

```diff
@@ -108,7 +110,7 @@ public AccountStatementExcelGenerationService(PropertyRepository propertyReposit
 			cell.setCellStyle(headerCellStyle);
 
 			cell = headerRow2.createCell(1);
-			cell.setCellValue("Vikas Nagar, Mauli Jagaran, UT Chandigarh");
+			cell.setCellValue(property.getColony() + ", UT Chandigarh");
```

Comment:
property.getColony() will have unlocalized value like `COLONY.KUMHAR`. The localized value can be obtained from localization.

Reply:
colony value picked from localization